static/js/components/GcnEventPage.jsx

```diff
@@ -252,28 +248,6 @@ const GcnEventPage = ({ route }) => {
     fetchGcnEvent(route.dateobs);
   }, [route, dispatch]);
 
-  useEffect(() => {
-    const fetchGcnEventSources = async (dateobs) => {
-      await dispatch(sourcesActions.fetchGcnEventSources(dateobs));
-    };
-    fetchGcnEventSources(route.dateobs);
-    dispatch(sourcesActions.fetchGcnEventSources(route.dateobs));
-  }, [route, dispatch]);
-
-  useEffect(() => {
-    const fetchGcnEventObservations = async (dateobs) => {
-      await dispatch(observationsActions.fetchGcnEventObservations(dateobs));
-    };
-    fetchGcnEventObservations(route.dateobs);
-  }, [route, dispatch]);
-
-  useEffect(() => {
-    const fetchGcnEventGalaxies = async (dateobs) => {
-      await dispatch(galaxiesActions.fetchGcnEventGalaxies(dateobs));
-    };
-    fetchGcnEventGalaxies(route.dateobs);
-  }, [route, dispatch]);
-
```

Comment:
This seems important but if you checked it works without all this stuff then it is fine with me.

Reply:
Yeah it just means we have to specifically ask for it in the selection form.